### PR TITLE
Return None if pm25_value is unsupported.

### LINKED ISSUE #45: pm25_value could be "unsupported" if dehumidifier didn't have pm2.5 sensor.
pm25_value could be "unsupported" if dehumidifier didn't have pm2.5 sensor.
Tested on RDI-640HH

### PR DESCRIPTION
**Pull request by RiANOl**
To fix #45

### CODE REVIEW

**Comment by qqaatw** [comment]
We need to fix this issue holistically not just the sensor in question, let me think about it.

**Comment by RiANOl** [comment]
Sure, take your time :)

File: custom_components/jcihitachi_tw/sensor.py [hunk 1]
```diff
@@ -107,6 +107,8 @@ def native_value(self):
         """Return the PM2.5 value."""
         status = self.hass.data[DOMAIN][UPDATED_DATA].get(self._thing.name, None)
         if status:
+            if status.pm25_value == "unsupported":
+                return None
             return status.pm25_value
```
Comment (qqaatw) [inline custom_components/jcihitachi_tw/sensor.py:112]: Please use one-line comprehension:
```suggestion
            return None if status.pm25_value == "unsupported" else status.pm25_value
```

**Review by qqaatw — COMMENTED** [review]
Left a comment. Can you please apply the change to these entities also:
- `JciHitachiPowerConsumptionSensorEntity`
- `JciHitachiIndoorHumiditySensorEntity`
- `JciHitachiIndoorTemperatureSensorEntity`

Thank you for your contribution.

File: custom_components/jcihitachi_tw/sensor.py (hunk 1 above)
Comment (RiANOl) [reply custom_components/jcihitachi_tw/sensor.py]: Fixed

**Review by qqaatw — APPROVED** [review]
LGTM

**Pull request merged**